src/theme/mod.rs

```diff
@@ -25,6 +25,7 @@ pub static FONT_AWESOME_WOFF: &'static [u8] = include_bytes!("_FontAwesome/fonts
 pub static FONT_AWESOME_WOFF2: &'static [u8] = include_bytes!("_FontAwesome/fonts/fontawesome-webfont.woff2");
 pub static FONT_AWESOME_OTF: &'static [u8] = include_bytes!("_FontAwesome/fonts/FontAwesome.otf");
 
+
```

Comment:
well this commit was unneeded ;)
It was enough to amend previous commit and push with force. Travis would pick it up again